examples/language-sdk-instrumentation/golang-push/rideshare-alloy/README.md

```diff
@@ -0,0 +1,50 @@
+# Rideshare Example with Alloy Profiling
+
+This example demonstrates how to use Alloy to receive and forward profiles from the rideshare example application.
+
+## Architecture
+
+- Regional services (us-east, eu-north, ap-south) send profiles to Alloy
+- Alloy receives profiles on port 9999 and forwards them to Pyroscope
+- Pyroscope stores and processes profiles
+- Grafana visualizes the profiling data
+
+## Configuration
+
+The example uses this Alloy configuration:
+```alloy
+pyroscope.receive_http "default" {
+    http {
+        listen_address = "0.0.0.0"
+        listen_port = 9999
+    }
+    forward_to = [pyroscope.write.backend.receiver]
+}
+
+pyroscope.write "backend" {
+    endpoint {
+        url = "http://pyroscope:4040"
+    }
+    external_labels = {
```

Comment:
In some other examples we add a commented out snipped of how to configure basic auth for cloud 
https://github.com/grafana/pyroscope/blob/main/examples/grafana-agent-auto-instrumentation/ebpf/local/config.river#L47-L51
It may be worth adding here and in the config itself as well.